docs/fiddles/features/drag-and-drop/preload.js

```diff
@@ -0,0 +1,11 @@
+const { contextBridge, ipcRenderer } = require('electron')
+const fs = require('fs')
+
+contextBridge.exposeInMainWorld('electron', {
+  startDrag: (fileName) => {
+    // Create a new file to copy - you can also copy existing files.
+    fs.writeFileSync(fileName, '# Test drag and drop')
+
+    ipcRenderer.send('ondragstart', process.cwd() + `/${fileName}`)
+  }
```

Comment:
I think we should use the async version here so they are of to a good start. What do you think @tonyferrell, @erickzhao ?

```suggestion
const fs = require('fs').promises;

contextBridge.exposeInMainWorld('electron', {
  startDrag: async (fileName) => {
    // Create a new file to copy - you can also copy existing files.
    await fs.writeFile(fileName, '# Test drag and drop')

    ipcRenderer.send('ondragstart', process.cwd() + `/${fileName}`)
  }
```

Reply:
Oh yeah, great idea. :)